spec/features/gym_search/index_spec.rb

```diff
@@ -1,7 +1,10 @@
 require 'rails_helper'
 
-RSpec.describe 'gyms near me page' do
+RSpec.describe 'gyms near me page', :vcr do
   before do
+    allow(BackEndFacade).to receive(:get_user_friends).with(@user.id).and_return([])
+    allow(BackEndFacade).to receive(:get_user_gyms).with(@user.id).and_return([])
+    allow(BackEndFacade).to receive(:get_user_events).with(@user.id).and_return([])
```

Comment:
These stubs needed to be added to get the gym search index spec to pass.

Reply:
Adding on my end as we speak